draft-ietf-tls-subcerts.md

```diff
@@ -327,15 +332,16 @@ This document defines the following extension code point.
    } ExtensionType;
 ~~~~~~~~~~
 
+### Server authentication
+
 A client which supports this specification SHALL send an empty
-"delegated_credential" extension in its ClientHello.  If the client receives a
-delegated credential without indicating support, then the client MUST abort with
-an "unexpected_message" alert.
+"delegated_credential" extension in its ClientHello.  If the server receives a
```

Comment:
do we need this sentence here? "If the server receives a delegated credential without indicating support, then the server MUST abort.." I think this is a sentence for client authentication.

Reply:
The client and server are mixed up here. Fixing.